implot-examples/implot-wgpu-demo/Cargo.toml

```diff
@@ -3,14 +3,15 @@ name = "implot-wgpu-demo"
 version = "0.1.0"
 authors = ["Benedikt Mandelkow <benedikt.mandelkow@rwth-aachen.de>", "imgui-wgpu contributors"]
 edition = "2018"
+resolver = "2"
 
 [dependencies]
 implot = { path = "../../" }
 
-wgpu = "=0.6.0"
-winit = "0.23" # opening windows and handling input
+wgpu = "0.10"
```

Comment:
I pinned some of these exactly because I've seen breakage between minor versions - I don't remember if wgpu was one of the crates that were affected by this though. I'm OK with removing the `=` for it for now, if we run into trouble we can always re-add it.